services/ui-src/src/forms/mcpar/mcpar.json

```diff
@@ -2983,12 +2983,36 @@
                 }
               },
               {
-                "id": "sanction_remediationDate",
-                "type": "date",
-                "validation": "date",
+                "id": "sanction_remediationCorrected",
+                "type": "radio",
+                "validation": "radio",
                 "props": {
                   "label": "D3.VIII.8 Remediation date non-compliance was corrected",
-                  "hint": "What day was it remediated, i.e. When was the non-compliance corrected? Select a date."
+                  "hint": "Was the non-compliance remediated?",
+                  "choices": [
+                    {
+                      "id": "J9FTVN9fJpgXX6X2xJiAiP",
+                      "label": "Yes",
+                      "children": [
+                        {
+                          "id": "sanction_remediationCorrectedDate",
```

Comment:
Blocking: This id should remain `sanction_remediationDate`. This is because there are already reports in the database that have data for this field id and if it changes we have to handle for both, so anywhere we reference it in the codebase it would have to be `fieldAValue || fieldBValue`, which could get out of hand quickly. Changing each instance of the following across the four files will fix it:

- `sanction_remediationCorrectedDate` goes back to `sanction_remediationDate`
- `remediationCorrectedDate` goes back to `remediationDate`